interactions/client/bot.py

```diff
@@ -75,21 +77,25 @@ class Client:
     def __init__(
         self,
         token: str,
+        cache_limits: Optional[Dict[type, int]] = None,
+        intents: Intents = Intents.DEFAULT,
+        shards: Optional[List[Tuple[int]]] = None,
+        default_scope=None,  # todo typehint
```

Comment:
```suggestion
        default_scope: Union[int, Snowflake, List[int, Snowflake]] = None,
```
that would work, right?